CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Version History
 
+
+#### 20150210
```

Comment:
Not sure exactly what the existing convention is, but it might be better to reserve changes in the build version specifically for when there are functional changes to the project beyond adding support for new node releases. I think(?) that's what ruby-build does with their releases.

Otherwise we should also change the actual [build version](https://github.com/OiNutter/node-build/blob/df67cec08280250a64fb29e7b4380af6a4d95038/bin/node-build#L3) as well.